src/tracing/utils.ts

```diff
@@ -2,7 +2,8 @@
  * Returns the time elapsed between page load and now.
  * @param markName the identifier for the performance mark to be created and measured.
  */
-export function calculateElapsedTimeWithPerformanceMark(markName: string): number {
+export function calculateElapsedTimeWithPerformanceMark(markName: string): number | undefined {
   const elapsedTime = performance.mark(markName)
+  if (!elapsedTime) return undefined
```

Comment:
This is fine as a hotfix, but we are cutting a section of browsers out of our metrics by not polyfilling this. Do we want to find some other way to get at an equivalent value?

Reply:
@zzmp yes we most likely do. i think it's fine to fix the crash now and follow up with a fix to get the metrics for those iOS browsers. right now, the iOS browsers aren't reporting metrics anyway so this just helps us on the sentry side

Reply:
sgtm - please just make sure the linear ticket stays open :)